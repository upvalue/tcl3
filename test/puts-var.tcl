set x 42
puts "forty-two $x"
