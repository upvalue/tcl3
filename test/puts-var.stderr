{"type": "TK_ESC", "begin": 0, "end": 3, "body": "set"}
{"type": "TK_SEP", "begin": 3, "end": 4, "body": " "}
{"type": "TK_ESC", "begin": 4, "end": 5, "body": "x"}
{"type": "TK_SEP", "begin": 5, "end": 6, "body": " "}
{"type": "TK_ESC", "begin": 6, "end": 8, "body": "42"}
{"type": "TK_EOL", "begin": 8, "end": 9, "body": "\n"}
{"type": "TK_ESC", "begin": 9, "end": 13, "body": "puts"}
{"type": "TK_SEP", "begin": 13, "end": 14, "body": " "}
{"type": "TK_ESC", "begin": 15, "end": 25, "body": "forty-two "}
{"type": "TK_VAR", "begin": 26, "end": 27, "body": "x"}
{"type": "TK_ESC", "begin": 27, "end": 27, "body": ""}
{"type": "TK_EOL", "begin": 28, "end": 29, "body": "\n"}
{"type": "TK_EOF", "begin": 28, "end": 29, "body": "\n"}
